From 956cbf01e345779d869490b17c31e0fdb7014d17 Mon Sep 17 00:00:00 2001
From: Alistair Delva <adelva@google.com>
Date: Tue, 28 Jan 2020 12:35:12 -0800
Subject: [PATCH 619/830] ANDROID: gki_defconfig: Enable CONFIG_BTT
X-NVConfidentiality: public

Without this, the virtio_pmem driver does not seem to work.

Bug: 146400078
Change-Id: I2689aa4739b83a0a5a0e9930dc50b57e4f612525
Signed-off-by: Alistair Delva <adelva@google.com>
---
 arch/arm64/configs/gki_defconfig | 1 -
 arch/x86/configs/gki_defconfig   | 1 -
 2 files changed, 2 deletions(-)

diff --git a/arch/arm64/configs/gki_defconfig b/arch/arm64/configs/gki_defconfig
index ebfe18c5950b..145eaa767d86 100644
--- a/arch/arm64/configs/gki_defconfig
+++ b/arch/arm64/configs/gki_defconfig
@@ -398,7 +398,6 @@ CONFIG_ANDROID=y
 CONFIG_ANDROID_BINDER_IPC=y
 CONFIG_LIBNVDIMM=y
 # CONFIG_ND_BLK is not set
-# CONFIG_BTT is not set
 CONFIG_INTERCONNECT=y
 CONFIG_EXT4_FS=y
 CONFIG_EXT4_FS_SECURITY=y
diff --git a/arch/x86/configs/gki_defconfig b/arch/x86/configs/gki_defconfig
index a0a6978c1dd9..faa8a73528e2 100644
--- a/arch/x86/configs/gki_defconfig
+++ b/arch/x86/configs/gki_defconfig
@@ -334,7 +334,6 @@ CONFIG_ANDROID=y
 CONFIG_ANDROID_BINDER_IPC=y
 CONFIG_LIBNVDIMM=y
 # CONFIG_ND_BLK is not set
-# CONFIG_BTT is not set
 CONFIG_EXT4_FS=y
 CONFIG_EXT4_FS_SECURITY=y
 CONFIG_F2FS_FS=y
-- 
2.17.1

